Refactor0 (#6)
chengchingwen/Pickle.jl: README.md

```

> 2.  framework for easily analyze and implement required translation methods. The unconstructable data will 

Reviewer: ```suggestion
2.  framework for easily analyze and implement required translation methods. For those data which are
```

> be stored in a `Defer` object which is similar to `Expr` but is mutable.

Reviewer: ```suggestion
not able to restore directly will be stored in a `Defer` object which is similar to `Expr` but is mutable.
```

```

> 2.  framework for easy analysis and implement required translation methods. The unconstructable data will 

Reviewer 2: ```suggestion
2.  framework for easily analyze and implement required translation methods. The unconstructable data will 
```

> store in a `Defer` object which is similar to `Expr` but is mutable.

Reviewer 2: ```suggestion
stored in a `Defer` object which is similar to `Expr` but is mutable.
```
